format/Schema.fbs

```diff
@@ -17,6 +17,11 @@
 
 /// Logical types, vector layouts, and schemas
 
+/// Format Version History.
+/// Version 1.0 - Forward and backwards compatibility guaranteed.
+/// Version 1.1 - Add Decimal256 (No format release).
+/// Version 1.2 (Pending)- Add Interval MONTH_DAY_NANO
```

Comment:
Did we actually formalize the notion of format versions?

Reply:
We [have](https://arrow.apache.org/docs/format/Versioning.html) this but I don't know if we've been officially announcing/voting on them as separate releases.